network/dag/tree/tree.go

```diff
@@ -101,29 +132,49 @@ func (t *tree) Load(leaves map[uint32][]byte) error {
 		if err != nil {
 			return err
 		}
-		err = t.updateOrCreatePath(split, func(n *node) error {
-			return n.data.Add(data)
+		t.updateOrCreatePath(split, func(n *node) {
+			n.data.Add(data)
```

Comment:
error is ignored?

Reply:
No error can occur here since `n.data` and `data` are of the same concrete type.

changed to explicitly ignore error.